miro_preprocessor/xml_to_json_converter/install_deps.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+pip install -r requirements.txt
```

Comment:
This script is never used.

Reply:
It's used by the test operation in the python container - this is again optionally present to allow the project to trigger installing dependencies for test and build in whatever way is necessary.

This and the `generate_tf_vars` hook should probably be documented somewhere.